src/ros_filter.cpp

```diff
@@ -1647,28 +1675,29 @@ namespace RobotLocalization
       integrateMeasurements(curTime);
 
       // Get latest state and publish it
-      nav_msgs::Odometry filteredPosition;
+      nav_msgs::Odometry odomMsg;
```

Comment:
As above: if we don't have to change this, I'd prefer that we leave it alone, even if the variable name is admittedly not great. This rename ends up creating a lot of changed lines in the PR that don't actually change any functionality.